src/cache.js

```diff
@@ -1,18 +1,24 @@
 import { shouldInvokePromise, calculateExpiry } from './utils';
 
-const createCache = (promiseFunc, options = {}) => {
-  const cache = {};
+const cache = {};
 
+const createCache = (promiseFunc, options = {}) => {
   const internalOptions = { timeToLive: calculateExpiry(options.timeToLive) };
 
-  async function read({ key, forceInvoke = false }, ...parameters) {
-    let response = cache[key];
-    if (shouldInvokePromise(cache, key, internalOptions.timeToLive, forceInvoke)) {
-      response = await promiseFunc(...parameters);
-      cache[key] = response;
-      internalOptions.timeToLive = calculateExpiry(options.timeToLive);
-    }
-    return response;
+  function read({ key, forceInvoke = false }, ...parameters) {
```

Comment:
Seeing as line _5_ is an arrow function, should this one also be?

Reply:
Last question left.

I think it should be good as an arrow function as it has an immediate return.